craft_application/services/service_factory.py

```diff
@@ -157,11 +156,11 @@ def set_kwargs(
     ) -> None:
         """Set up the keyword arguments to pass to a particular service class.
 
-        PENDING DEPRECATION: use update_kwargs instead
+        DEPRECATED: use update_kwargs instead
```

Comment:
This comes from a separate commit that I'll be PRing separately. Didn't mean to bring it in here, but I doubt this will land before it does.

Reply:
https://github.com/canonical/craft-application/pull/660